elliott/elliottlib/cli/konflux_release_cli.py

```diff
@@ -110,7 +110,7 @@ async def run(self):
         release_name = f"ose-{major}-{minor}-{self.release_env}-{get_utc_now_formatted_str()}"
 
         env_config: ShipmentEnv = getattr(config.shipment.environments, self.release_env)
-        if env_config.shipped():
+        if env_config.shipped() and self.release_env == "prod":
```

Comment:
nice! can you raise error instead of warn when env is prod? that would be a good safeguard

Reply:
updated